lib/rspecq/reporter.rb

```diff
@@ -116,16 +116,18 @@ def humanize_duration(seconds)
     def flaky_jobs_to_sentry(jobs, build_duration)
       return if jobs.empty?
 
-      Raven.capture_message("Flaky jobs detected", level: "warning", extra: {
-        build: @build_id,
-        build_timeout: @timeout,
-        queue: @queue.inspect,
-        object: self.inspect,
-        pid: Process.pid,
-        flaky_jobs: jobs,
-        flaky_jobs_count: jobs.count,
-        build_duration: build_duration
-      })
+      jobs.each do |job|
+        Raven.capture_message("[Flaky] #{job}", level: "warning", extra: {
```

Comment:
How about using a more user-friendly title, like:

> Flaky test: spec/controllers/foo_controller_spec.rb[1:3]

Reply:
2a71d84 (with the spec file removed)